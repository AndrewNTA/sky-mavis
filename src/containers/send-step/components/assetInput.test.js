import React from 'react';
import renderer from 'react-test-renderer';
import AssetInput from './assetInput';

test('SendStep should render asset input', () => {
  const component = renderer.create(
    <AssetInput
      selectedCurrency= {{
        primaryCurrency: 'USD',
        icon: 'icon_url'
      }}
      onOpenModal={() => {}}
    />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});