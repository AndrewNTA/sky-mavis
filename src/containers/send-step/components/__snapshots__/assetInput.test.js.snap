// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendStep should render asset input 1`] = `
<div
  className="r-asset-input-container"
>
  <div
    className="r-asset-info"
  >
    <img
      alt="USD"
      className="r-asset-currency-icon"
      src="icon_url"
    />
    <span
      className="r-asset-currency-text"
    >
      USD
    </span>
  </div>
  <img
    alt="options"
    className="r-asset-option-icon"
    onClick={[Function]}
    src="option-icon.png"
  />
</div>
`;
